jdaviz/configs/imviz/tests/test_simple_aper_phot.py

```diff
@@ -489,3 +490,26 @@ def test_curve_of_growth(with_unit):
     with pytest.raises(TypeError, match='Unsupported aperture'):
         _curve_of_growth(data, cen, EllipticalAnnulus(cen, 3, 8, 5), 100,
                          pixarea_fac=pixarea_fac)
+
+
+def test_cubeviz_batch(cubeviz_helper, spectrum1d_cube_fluxunit_jy_per_steradian):
+    cubeviz_helper.load_data(spectrum1d_cube_fluxunit_jy_per_steradian, data_label='test')
+    subset_plugin = cubeviz_helper.plugins['Subset Tools']._obj
+    phot_plugin = cubeviz_helper.plugins['Aperture Photometry']._obj
+    uc_plugin = cubeviz_helper.plugins['Unit Conversion']
+
+    fv = cubeviz_helper.app.get_viewer('flux-viewer')
+    fv.apply_roi(CircularROI(xc=5, yc=5, radius=2))

```

Comment:
Why not use the load region API?

Reply:
I'm running into test failures after switching to `load_regions`, trying to figure out if they're real or I'm calling something incorrectly.